namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev/resources/reporting-landing-access.tf

```diff
@@ -22,7 +22,9 @@ data "aws_iam_policy_document" "reporting_access" {
 
     resources = [
       "arn:aws:s3:::eu-west-2-delius-mis-dev-dfi-extracts/dfinterventions/dfi/*",
-      "arn:aws:s3:::eu-west-2-delius-mis-dev-dfi-extracts/dfinterventions/dfi/"
+      "arn:aws:s3:::eu-west-2-delius-mis-dev-dfi-extracts/dfinterventions/dfi/",
+      "arn:aws:s3:::eu-west-2-delius-mis-dev-dfi-extracts/exports/csv/reports/*",
+      "arn:aws:s3:::eu-west-2-delius-mis-dev-dfi-extracts/exports/csv/reports/",
```

Comment:
The full path has to be `s3://eu-west-2-delius-prod-dfi-extracts/dfinterventions/dfi/csv/reports/` which is already included in the top two resource declarations